worker/jobProcessor.js

```diff
@@ -101,6 +110,13 @@ jobQueue.process(jobType.SAMPLE_TIMEOUT, (job, done) => {
   const dbStartTime = Date.now();
   sampleTimeoutJob.execute()
   .then((dbRes) => {
+    // send the timeoutsample to the client by publishing it to redis channel
+    if (dbRes && dbRes.timedOutSamples) {
+      dbRes.timedOutSamples.forEach((sample) => {
+        publisher.publishSample(sample, subHelper.model, sampleEvent.upd);
+      });
```

Comment:
So, sending 2 events for a sample timeout when worker process is enabled is fine?